src/pages/ProductDetail/DetailContainer/Custom.js

```diff
@@ -0,0 +1,274 @@
+          <TextColorBox
+            onClick={() => setFontColor('#FF0000')}
+            color="#FF0000"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#FF1493')}
+            color="#FF1493"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#FFB6C1')}
+            color="#FFB6C1"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#FFDAB9')}
+            color="#FFDAB9"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#F0E68C')}
+            color="#F0E68C"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#F5DEB3')}
+            color="#F5DEB3"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#98FB98')}
+            color="#98FB98"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#00FF00')}
+            color="#00FF00"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#00CED1')}
+            color="#00CED1"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#1E90FF')}
+            color="#1E90FF"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#4169E1')}
+            color="#4169E1"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#9370DB')}
+            color="#9370DB"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#F4A460')}
+            color="#F4A460"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#D2691E')}
+            color="#D2691E"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#D3D3D3')}
+            color="#D3D3D3"
+          />
+          <TextColorBox
+            onClick={() => setFontColor('#708090')}
+            color="#708090"
+          />
+        </TextColor>
+      </CustomContainer>
+    </OptionChoice>
+  );
+};
+
+const OptionChoice = styled.div`
+  margin: 8px 0;
+  background-color: white;
+  font-weight: 600;
+`;
+
+const OptionTilte = styled.div`
+  padding: 15px 3px 5px 3px;
+  font-size: 15px;
+`;
+
+const CustomContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 10px;
+  margin: 5px;
+  border: 1px solid #000;
+  border-radius: 4px;
+  box-shadow: 0 2px 2px rgba(0, 0, 0, 0.3);
+`;
+
+const CustomChoice = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 10px 0 20px 0;
+`;
+
+const CustomStyle = styled.div`
+  padding: 0 25px;
+  p {
+    padding-bottom: 2px;
+  }
+  ul {
+    display: flex;
+    flex-wrap: wrap;
+    width: 140px;
+  }
+`;
+
+const CustomLayoutBox = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 3px;
+  width: 36px;
+  height: 36px;
+  border: 2px solid #c3c3c3;
+  background: no-repeat center/50% url(${props => props.picture});
```

Comment:
```suggestion
  background: no-repeat center/50% url(${({picture})=> picture});
```
매개변수자리에서 구조분해 가능합니다.

Reply:
반영해주세요

Reply:
같은 내용으로 3번째 코멘트입니다

Reply:
수정했습니다!